src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/MerchandiseReturn/Blocks/form.html.twig

```diff
@@ -0,0 +1,90 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{{ form_start(merchandiseReturnForm, {action: path('admin_merchandise_returns_update', {'merchandiseReturnId': editableMerchandiseReturn.merchandiseReturnId.value})}) }}
+  <div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">person</i>
+    {{ 'Merchandise Return'|trans({}, 'Admin.Orderscustomers.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {{ form_errors(merchandiseReturnForm) }}
+      <div class="form-group row">
+        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
+          {{ 'Customer'|trans({}, 'Admin.Global') }}
+        </label>
+        <div class="col-sm form-control-value">
+          <p>{{ editableMerchandiseReturn.customerFullName }}</p>
+          <a href="{{ path('admin_customers_view', {'customerId': editableMerchandiseReturn.customerId.value }) }}" class="btn btn-outline-secondary">
+            {{ 'View details on the customer page'|trans({}, 'Admin.Actions') }}
```

Comment:
```suggestion
            {{ 'View details on the customer page'|trans({}, 'Admin.Orderscustomers.Help') }}
```

Reply:
@Junebyun, I added punctuation to keep all help texts/tooltips stay consistent. ;-)